src/services/pocket-relayer.ts

```diff
@@ -599,46 +608,31 @@ export class PocketRelayer {
     // Pocketjs session calls are more prone to timeouts when getting the dispatchers,
     // Doing the rpc call directly minimizes the possibily of failing due to timeouts
     const dispatcher = this.dispatchers[getRandomInt(0, this.dispatchers.length)]
-    const sessionHeaderRequestBody = {
+    const dispatchRequestBody = {
       app_public_key: appPublicKey,
       chain: blockchainID,
       session_height: 0,
     }
-    const sessionHeaderURL = `${dispatcher}v1/client/dispatch`
-    const sessionHeader = await axios.post(sessionHeaderURL, sessionHeaderRequestBody)
-
-    if (sessionHeader.status !== 200) {
-      throw new Error(`Error obtaining a session: ${sessionHeader.data}`)
-    }
-
-    // ----- TODO: To be Removed -----
-    // Pull the session so we can get a list of nodes and cherry pick which one to use
-    const pocketSession = await this.pocket.sessionManager.getCurrentSession(
-      pocketAAT,
-      blockchainID,
-      this.updateConfigurationTimeout(this.pocketConfiguration),
-      2
-    )
+    const dispatchURL = `${dispatcher}v1/client/dispatch`
+    const dispatchResponse = await axios.post(dispatchURL, dispatchRequestBody)
 
-    if (pocketSession instanceof Error) {
-      logger.log('error', 'ERROR obtaining a session: ' + pocketSession.message, {
+    if (dispatchResponse.status !== 200) {
+      logger.log('error', `ERROR: obtaining a session: ${dispatchResponse.data}`, {
         relayType: 'APP',
         typeID: application.id,
         origin: this.origin,
         blockchainID,
         requestID,
       })
 
-      return pocketSession
+      throw new Error(`Error obtaining a session: ${dispatchResponse.data}`)
     }
-    // ----- End of remove session -----
 
-    // Start the relay timer
-    const relayStart = process.hrtime()
+    const sessionHeader = new SessionHeader(appPublicKey, blockchainID, BigInt(0))
 
     // Converts the rpc response in a way that is compatible with pocketjs for
     // sending relays through
-    let nodes: Node[] = (sessionHeader.data.session.nodes as Node[]).map(
+    let nodes: Node[] = (dispatchResponse.data.session.nodes as Node[]).map(
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This mapping could be moved to a static function on the class or to `helpers.ts`